index.js

```diff
@@ -103,8 +103,8 @@ module.exports = {
       'components/dialog/dialog-theme.scss'
     ];
 
-    var pathBase = this.project.addonPackages['ember-paper'].path;
-    var angularMaterialPath =  'node_modules/angular-material-source/src';
+    var pathBase = this.project.nodeModulesPath;
```

Comment:
yep - sorry if i made that not so clear, my bad - that is the issue - npm 2.x was a recursive dep tree, in 3.x its flat